src/arch/Sound/RageSoundDriver_WaveOut.h

```diff
@@ -33,6 +32,7 @@ class RageSoundDriver_WaveOut: public RageSoundDriver
 	int m_iSampleRate;
 	bool m_bShutdown;
 	int m_iLastCursorPos;
+	bool b_InitSuccess = false;
```

Comment:
Initialize in constructor, not here.

Reply:
Now initialized here

https://github.com/itgmania/itgmania/blob/1a3ef9dbc57460762f7e6bbc15a001dcd6bc7061/src/arch/Sound/RageSoundDriver_WaveOut.cpp#L122-L124